app/models/task.rb

```diff
@@ -10,4 +10,8 @@ class Task < ApplicationRecord
   enum :status, { pending: 0, in_progress: 1, done: 2 }
 
   paginates_per 6
+
+  include PgSearch::Model
+  pg_search_scope :search_by_status, against: :status
```

Comment:
Let's support search by `task#title`. (or maybe body?)

Reply:
And I don't think we need pg search for status field. For `status` we can just go for `where(status: status)`.